ddpui/celery.py

```diff
@@ -5,7 +5,10 @@
 # Set the default Django settings module for the 'celery' program.
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "ddpui.settings")
 
-app = Celery("ddpui", backend="redis://localhost", broker="redis://localhost")
+REDIS_HOST = os.getenv("REDIS_HOST", "localhost")
+REDIS_PORT = int(os.getenv("REDIS_PORT", "6379"))
+
+app = Celery('ddpui', backend=f'redis://{REDIS_HOST}:{REDIS_PORT}', broker=f'redis://{REDIS_HOST}:{REDIS_PORT}')
```

Comment:
Could you add a comment on what the backend does and what the broker does

Reply:
Added